etc/httpd/global.conf

```diff
@@ -884,3 +884,6 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?shapeoftheweb(/?.*)$ /b/$1shapeoftheweb$2 [P
 
 # bug 1120658
 RewriteRule ^/seamonkey-transition\.html$ http://www-archive.mozilla.org/seamonkey-transition.html [L,R=301]
+
+# bug 1121082
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?hello$ /$1firefox/hello/ [L,R=301]
```

Comment:
probably want to change the end of the match to `hello/?$` so that URLs with a trailing slash are caught.